CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # Main (unreleased)
 
+- [BUGFIX] Statsd Integration: Adding logger instance to the statsd mapper instantiation. (@gaantunes)
```

Comment:
Hey, can you retain the grouping of FEATURE -> ENHANCEMENT -> BUGFIX -> CHANGE -> DEPRECATION? And then newer changeling entries get added to the bottom of the group, so they're sorted by group & time.